### PR TITLE
Adding Serverless + Golang + AL2 + CodeSee example

### PR DESCRIPTION
**Pull request by kien-cs**
This will add a new example to show customer on how to integrate with CodeSee using Serverless Framework + Golang + AL2.

### CODE REVIEW

File: serverless-al2-example/Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
```
Comment (jleven) [inline serverless-al2-example/Gopkg.toml:20]: Are these comments meant to be here?

File: serverless-al2-example/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  bootstrap:
+    handler: hello-src/main.go
+    layers:
+      - arn:aws:lambda:us-east-1:666523192759:layer:codesee-dd-bridge:6 # This version currently only support x86, no ARM support yet.
+    environment:
+      DD_APM_DD_URL: http://127.0.0.1:8080
+      DD_APM_NON_LOCAL_TRAFFIC: true
+      DD_APM_ENABLED: true
+
+      CODESEE_ENABLE_ACCESS_LOGS: true # DEBUG - turn off in Prod
```
Comment (jleven) [inline serverless-al2-example/serverless.yml:21]: I doubt everyone will read things carefully. Maybe we set this to `false` but leave a comment to "turn on for debug logging for codesee layer"

File: serverless-al2-example/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Set required environment variables for CodeSee:
+
+```
+DD_APM_DD_URL: http://127.0.0.1:8080
+DD_APM_NON_LOCAL_TRAFFIC: true
+DD_APM_ENABLED: true
+CODESEE_BRIDGE_FORWARD_HOST: https://in-datadog.codesee.io
+CODESEE_BRIDGE_TOKEN: <YOUR_CODESEE_TOKEN>
+```
```
Comment (jleven) [inline serverless-al2-example/README.md:21]: Can you add a little more information here?

Should these be set in certain files of the example? Or set via some other method? I feel like I've seen several of these in the code. If they are already set in the code, why are we also listing them here?

Comment (jleven) [reply serverless-al2-example/README.md:21]: I think you're telling the user to make sure these are set properly in the serverless.yml, is that right?

File: serverless-al2-example/Gopkg.toml (hunk 1 above)
Comment (kien-cs) [reply serverless-al2-example/Gopkg.toml:20]: nope will remove

File: serverless-al2-example/README.md (hunk 3 above)
Comment (kien-cs) [reply serverless-al2-example/README.md:21]: yea I'll add the note: Make sure the following environment variables are set in your `serverless.yml` file.

File: serverless-al2-example/serverless.yml (hunk 2 above)
Comment (kien-cs) [reply serverless-al2-example/serverless.yml:21]: will flip around

**Review by jleven — APPROVED** [review]
Awesome work!

**Pull request merged**